### PR TITLE
elstoc's edits to the release notes.

### PR DESCRIPTION
**Pull request by paperdigits**
Just a few more edits. :D

### CODE REVIEW

**Comment by elstoc** [comment]
Thanks @paperdigits

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ making a backup is strongly advised.
 
 ## New Features And Changes
 
-- Add support for curved gradients. This can be helpful when putting a gradient mask on an  image with a horizon line that is curved because lens distortion. This can also be used towards artistic goals.
+- Add support for curved gradients. This can be helpful when putting a gradient mask on an image with a horizon line that is curved due to lens distortion. This can also be for towards artistic goals.
```
Comment (elstoc) [inline RELEASE_NOTES.md:112]: please can you remove the word "towards"

Comment (paperdigits) [reply RELEASE_NOTES.md]: yes, done.

**Comment by TurboGit** [comment]
Thanks!

**Pull request merged**